src/main/java/sirius/biz/jobs/batch/file/XMLImportJob.java

```diff
@@ -27,6 +47,93 @@
      */
     protected XMLImportJob(XMLImportJobFactory factory, ProcessContext process) {
         super(factory.fileParameter, process);
+        requireValidFile = process.getParameter(factory.requireValidFile).orElse(false);
+        xsdResourcePath = factory.getXsdResourcePath();
```

Comment:
Is therre a use case for having a validation on xml file import and ignoring the result? If not, then why not just use the xsdPath parameter to determine if a validation should happen?

Reply:
The path dictates if a validation should happen. The flag controls the behavior of the job:
- Import: Abort or try to import anyway.
- Export: finish the job with either failed or success.
In both cases, validation errors will be logged accordingly as warning/error.

Reply:
We are inside an import job here. If you have a use case for validation and ignoring the results, than that is fine. Just I don't see that use case :-)

Reply:
Fine for me